sdk/appconfiguration/app-configuration/package.json

```
@@ -60,10 +60,10 @@
   "sideEffects": false,
   "autoPublish": false,
   "dependencies": {
-    "@azure/core-asynciterator-polyfill": "1.0.0-preview.1",
-    "@azure/core-http": "1.0.0-preview.6",
-    "@azure/core-paging": "1.0.0-preview.2",
-    "@azure/core-tracing": "1.0.0-preview.5",
+    "@azure/core-asynciterator-polyfill": "^1.0.0",
+    "@azure/core-http": "^1.0.0",
+    "@azure/core-paging": "^1.0.0",
+    "@azure/core-tracing": "^1.0.0-preview.5",
```

I think the reason behind pinning to a specific preview version was because breaking changes can be introduced between versions. Since core-tracing isn't GA yet we should probably stick to pinning to a specific version.

---

Preview versions should be pinned

```suggestion
    "@azure/core-tracing": "1.0.0-preview.5",
```

---

Got it.